### DISCUSSION TITLE
full list of usable variables in arch_config

### DISCUSSION THREAD

**Discussion by AslantheAslan** [timeseriesAI/tsai#678] [Q&A] [answered]
I have been using tsai for a while now, but I still don't know all variables that we can pass to arch_config. For example, how can I define an LSTM structure that has 2 layers, where the first layer's size is [128,128] and the second layer's size is [128,2]?

**Comment by oguiza** [comment, marked as answer]
Hi @AslantheAslan,
There's no list of arguments that can be passed to models in `tsai` as each one of them allows different ones.
I'd recommend you take a look at the documentation for each architecture to understand what can be used. 
Here's the [link](https://timeseriesai.github.io/tsai/models.rnnplus.html#lstmplus) for LSTMPlus (it's an enhanced version of LSTM that allows more arguments).

**Reply by AslantheAslan** [reply]
Thanks for the response. I have overlooked LSTMPlus until now, I will definitely utilize its different arguments.
